mu-plugins/10up-plugin/package.json

```diff
@@ -15,13 +15,12 @@
     "node": ">=12.0.0"
   },
   "devDependencies": {
-    "10up-toolkit": "^2.2.0-alpha.6"
+    "10up-toolkit": "^2.2.0-alpha.7"
```

Comment:
I think this should only get merged once the actual 2.2.0 version has been released

Reply:
Yep, I just wanted to get this reviewed ahead of final release.